<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
    <property name="verticalSpacing">
     <number>7</number>
    </property>
    <item row="0" column="2" rowspan="15">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="checkBox_testknoepfe">
      <property name="text">
       <string>Test-Knöpfe zeigen</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButton_dijkstra">
      <property name="text">
       <string>Weg zwischen zwei Städten finden</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_add_city">
      <property name="text">
       <string>Add City</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="pushButton_teste_was">
      <property name="toolTip">
       <string>Testet, ob Ihre Eingabe eine Zahl oder ein Text ist</string>
      </property>
      <property name="text">
       <string>Teste was</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QPushButton" name="pushButton_test_dijkstra">
      <property name="text">
       <string>Test Dijkstra</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_eingabe">
      <property name="text">
       <string>Ihre Eingabe</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QPushButton" name="pushButton_test_abstract_map">
      <property name="text">
       <string>Test Abstract Map</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_teste_was">
      <property name="placeholderText">
       <string>Ihre Eingabe</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pushButton_test_draw_city">
      <property name="text">
       <string>Test Draw City</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_fill_map">
      <property name="text">
       <string>Fill Map</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Test Map</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="pushButton_test_street">
      <property name="text">
       <string>Teste Street</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton_add_street">
      <property name="text">
       <string>Add Street</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuInfo">
     <property name="title">
      <string>Info</string>
     </property>
     <addaction name="actionAbout"/>
    </widget>
    <addaction name="actionOPen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="actionClear_Scene"/>
    <addaction name="separator"/>
    <addaction name="menuInfo"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOPen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
